FrontBackMatter/ExploringInhibition.tex

```diff
@@ -762,6 +840,24 @@ \subsection{Feature tuning and inhibition}
 shown, we could mirror the effects of decreasing the contrast,
 matching results found by \cite{Nienborg2013} in mouse visual cortex.
 
+It is worth noting that the average magnitude of contrast dependent
+size-tuning shifts has not changed significantly between the SCAL and
+SEPI models. However there is far greater diversity observed between
+different neurons than was present in the simpler SCAL model. This
+diversity can also be observed in experiments, suggesting something
+about the SEPI model captures this phenomenon more accurately. This
+increase in diversity is likely the result of greater diversity in
+connectivity, due to the separation of excitatory and inhibitory
+neurons into distinct cell classes neurons can now receive varying
+levels of excitation and inhibition. If we accept the argument made in
+the previous chapter and put forth by \cite{Cavanaugh2002} and
+\cite{Solomon2006}, that the shift in size tuning emerges as a result
+of different excitatory and inhibitory gain, this explanation perfect
+sense. Since neurons now have greater diversity in the excitatory and
+inhibitory inputs they receive the ratio between these components also
+becomes more diverse resulting in greater variance in the observed
+size-tuning shift.
```

Comment:
You sure don't like commas!  They are really needed here, after "ratio" and "diverse".